### PR TITLE
Peacemaker by piecemaker

### PR DESCRIPTION
**Pull request by SirPiecemaker**
Closed due to being merged in #33683

### CODE REVIEW

**Comment by LaVeyanFiend** [comment]
You may want to trim the summary line down. I've been told keep it to 1 line maximum.

> "Colt Peacemaker from realguns ported to main as it is one of the most popular 6-shooters to this day"

> **Mainline Colt Peacemaker from RealGuns**

File: data/json/items/gun/45colt.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "colt_saa",
+    "copy-from": "pistol_revolver",
+    "type": "GUN",
+    "name": "Colt Peacemaker",
+    "name_plural": "Colt Peacemaker",
+    "//": "Can be vaguely either an original or a reproduction, price based off of MSRP of Uberti Single-Action Cattleman.",
+    "description": "A third generation reproduction of the legendary Single Action Army, the Colt Peacemaker is one of the first revolvers to use a modern self-contained cartridge.  Made famous by westerns, it is still inn demand for Cowboy Action Shooting, enthusiasts and collectors, and some deem them the greatest gun ever made.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/45colt.json]:
```suggestion
    "description": "A third generation reproduction of the legendary Single Action Army, the Colt Peacemaker is one of the first revolvers to use a modern self-contained cartridge.  Made famous by westerns, it is still in demand for Cowboy Action Shooting, enthusiasts and collectors, and some deem them the greatest gun ever made.",
```

**Comment by Night-Pryanik** [comment]
Why don't you say anything in the OP about adding a new profession in addition to mainlining the gun?

**Comment by AMurkin** [comment]
First commits are from the #33629.

File: data/json/items/gun/45colt.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "colt_saa",
+    "copy-from": "pistol_revolver",
+    "type": "GUN",
+    "name": "Colt Peacemaker",
```
Comment (tenmillimaster) [inline data/json/items/gun/45colt.json]:
```suggestion
    "name": "Uberti Cattleman",
```

File: data/json/items/gun/45colt.json (hunk 1 above)
Comment (tenmillimaster) [reply data/json/items/gun/45colt.json]:
```suggestion
    "description": "This 7.5\" barreled Uberti Cattleman is a modern reproduction of the legendary Colt Single Action Army, one of the first revolvers to use a modern self-contained cartridge.  Made famous by westerns, the \"Peacemaker\" is still in demand for Cowboy Action Shooting, reenactors and collectors.  Unlike modern revolvers, the cylinder cannot swing out for loading, and spent brass must be ejected one at a time.",
```

File: data/json/items/gun/45colt.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "name_plural": "Colt Peacemaker",
```
Comment (tenmillimaster) [inline data/json/items/gun/45colt.json]:
```suggestion
    "name_plural": "Uberti Cattleman",
```

**Comment by tenmillimaster** [comment]
As Amurkin mentioned, you've got commits from 33629 in this PR, which means you either need to pull them out of this one, or close that PR and slightly resummarize this one.

**Pull request closed without merging**